docs/tutorials/using-backstage-proxy-within-plugin.md

```diff
@@ -36,10 +36,11 @@ and you want to be able to access it within backstage at
 
 ```yaml
 proxy:
-  '/<your-proxy-uri>':
-    target: https://api.myawesomeservice.com/v1
-    headers:
-      X-Custom-Source: backstage
+  endpoints:
```

Comment:
the config docs [here](https://backstage.io/docs/plugins/proxying/#configuration) include an endpoints level